src/views/Vacancies/PageView.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+
+import { FilterSidebar, FilterTagsHeader, VacancyList } from "./components";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: "flex",
+  },
+  sidebar: {
+    flex: "none",
+    width: "18rem",
+    minHeight: "100%",
+  },
+  content: {
+    flex: 1,
+    maxWidth: "48rem",
+    margin: "1.5rem auto",
+    minHeight: "100%",
+  },
+}));
+
+interface PageProps {
+  itemsLoading: boolean;
+  hasMoreItems: Boolean;
+  loadMoreItems: Function;
+  itemsList: Array<any>;
```

Comment:
some strong typing would be great. ensures we don't make silly mistake

Reply:
Done.

https://github.com/vonq/saleor-storefront/blob/dce4242590710005d34bd31f59a352c647c87266/src/views/Vacancies/PageView.tsx#L28

https://github.com/vonq/saleor-storefront/blob/dce4242590710005d34bd31f59a352c647c87266/src/views/Vacancies/components/VacancyCard.tsx#L43-L47